--- Makefile.old	2016-03-18 09:03:22.000000000 -0400
+++ ./Makefile	2018-01-11 17:26:53.823910095 -0500
@@ -1,10 +1,9 @@
 name = dstat
 version = $(shell awk '/^Version: / {print $$2}' $(name).spec)
 
-prefix = /usr
-sysconfdir = /etc
-bindir = $(prefix)/bin
-datadir = $(prefix)/share
+sysconfdir = $(PREFIX)/etc
+bindir = $(PREFIX)/bin
+datadir = $(PREFIX)/share
 mandir = $(datadir)/man
 
 .PHONY: all install docs clean
