--- fs/src/Makevars.orig	2023-06-29 15:51:28.000000000 -0400
+++ fs/src/Makevars	2023-11-14 08:36:54.005159721 -0500
@@ -1,14 +1,17 @@
 UNAME := $(shell uname)
-
-LIBUV := libuv-1.44.2
-
-PKG_LIBS = ./$(LIBUV)/.libs/libuv.a -pthread
+HOST := 
+UNAMEO := $(shell uname -o)
 
 SOURCES = $(wildcard *.cc unix/*.cc)
 OBJECTS = $(SOURCES:.cc=.o)
 
+PKG_LIBS += -luv
+
 ifeq ($(UNAME), SunOS)
 PKG_LIBS += -lkstat -lsendfile -lsocket -lxnet
+	 ifeq ($(UNAMEO), Android)
+	 HOST = --host=$(shell clang -print-target-triple)
+	 endif
 OBJECTS +=  bsd/setmode.o bsd/strmode.o bsd/reallocarray.o
 endif
 ifeq ($(UNAME), FreeBSD)
@@ -21,24 +24,12 @@
 OBJECTS +=  bsd/setmode.o bsd/strmode.o bsd/reallocarray.o
 endif
 
-PKG_CPPFLAGS = -I./$(LIBUV)/include -I. -pthread
-
 all: $(SHLIB)
 
-$(SHLIB): $(LIBUV)/.libs/libuv.a
-
-$(LIBUV)/Makefile:
-	(cd $(LIBUV) \
-	&& CC="$(CC)" CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS) $(CPICFLAGS) $(C_VISIBILITY) -std=c99" AR="$(AR)" RANLIB="$(RANLIB)" LDFLAGS="$(LDFLAGS)" ./configure --quiet)
-
-$(LIBUV)/.libs/libuv.a: $(LIBUV)/Makefile
-	$(MAKE) --directory=$(LIBUV) \
-		HAVE_DTRACE=0
-
 .PHONY: shlib-clean clean
 
 # shlib-clean: clean
 
 clean:
-	$(MAKE) --directory=$(LIBUV) distclean
+	$(MAKE) distclean
 	rm -f $(OBJECTS)
